// es-abstract/operations/2023.js
'use strict';

module.exports = {
	abs: {
		url: 'https://262.ecma-international.org/14.0/#eqn-abs'
	},
	AddEntriesFromIterable: {
		url: 'https://262.ecma-international.org/14.0/#sec-add-entries-from-iterable'
	},
	AddRestrictedFunctionProperties: {
		url: 'https://262.ecma-international.org/14.0/#sec-addrestrictedfunctionproperties'
	},
	AddToKeptObjects: {
		url: 'https://262.ecma-international.org/14.0/#sec-addtokeptobjects'
	},
	AddWaiter: {
		url: 'https://262.ecma-international.org/14.0/#sec-addwaiter'
	},
	AdvanceStringIndex: {
		url: 'https://262.ecma-international.org/14.0/#sec-advancestringindex'
	},
	'agent-order': {
		url: 'https://262.ecma-international.org/14.0/#sec-agent-order'
	},
	AgentCanSuspend: {
		url: 'https://262.ecma-international.org/14.0/#sec-agentcansuspend'
	},
	AgentSignifier: {
		url: 'https://262.ecma-international.org/14.0/#sec-agentsignifier'
	},
	AllocateArrayBuffer: {
		url: 'https://262.ecma-international.org/14.0/#sec-allocatearraybuffer'
	},
	AllocateSharedArrayBuffer: {
		url: 'https://262.ecma-international.org/14.0/#sec-allocatesharedarraybuffer'
	},
	AllocateTypedArray: {
		url: 'https://262.ecma-international.org/14.0/#sec-allocatetypedarray'
	},
	AllocateTypedArrayBuffer: {
		url: 'https://262.ecma-international.org/14.0/#sec-allocatetypedarraybuffer'
	},
	ApplyStringOrNumericBinaryOperator: {
		url: 'https://262.ecma-international.org/14.0/#sec-applystringornumericbinaryoperator'
	},
	ArrayCreate: {
		url: 'https://262.ecma-international.org/14.0/#sec-arraycreate'
	},
	ArraySetLength: {
		url: 'https://262.ecma-international.org/14.0/#sec-arraysetlength'
	},
	ArraySpeciesCreate: {
		url: 'https://262.ecma-international.org/14.0/#sec-arrayspeciescreate'
	},
	AsyncBlockStart: {
		url: 'https://262.ecma-international.org/14.0/#sec-asyncblockstart'
	},
	AsyncFromSyncIteratorContinuation: {
		url: 'https://262.ecma-international.org/14.0/#sec-asyncfromsynciteratorcontinuation'
	},
	AsyncFunctionStart: {
		url: 'https://262.ecma-international.org/14.0/#sec-async-functions-abstract-operations-async-function-start'
	},
	AsyncGeneratorAwaitReturn: {
		url: 'https://262.ecma-international.org/14.0/#sec-asyncgeneratorawaitreturn'
	},
	AsyncGeneratorCompleteStep: {
		url: 'https://262.ecma-international.org/14.0/#sec-asyncgeneratorcompletestep'
	},
	AsyncGeneratorDrainQueue: {
		url: 'https://262.ecma-international.org/14.0/#sec-asyncgeneratordrainqueue'
	},
	AsyncGeneratorEnqueue: {
		url: 'https://262.ecma-international.org/14.0/#sec-asyncgeneratorenqueue'
	},
	AsyncGeneratorResume: {
		url: 'https://262.ecma-international.org/14.0/#sec-asyncgeneratorresume'
	},
	AsyncGeneratorStart: {
		url: 'https://262.ecma-international.org/14.0/#sec-asyncgeneratorstart'
	},
	AsyncGeneratorUnwrapYieldResumption: {
		url: 'https://262.ecma-international.org/14.0/#sec-asyncgeneratorunwrapyieldresumption'
	},
	AsyncGeneratorValidate: {
		url: 'https://262.ecma-international.org/14.0/#sec-asyncgeneratorvalidate'
	},
	AsyncGeneratorYield: {
		url: 'https://262.ecma-international.org/14.0/#sec-asyncgeneratoryield'
	},
	AsyncIteratorClose: {
		url: 'https://262.ecma-international.org/14.0/#sec-asynciteratorclose'
	},
	AsyncModuleExecutionFulfilled: {
		url: 'https://262.ecma-international.org/14.0/#sec-async-module-execution-fulfilled'
	},
	AsyncModuleExecutionRejected: {
		url: 'https://262.ecma-international.org/14.0/#sec-async-module-execution-rejected'
	},
	AtomicReadModifyWrite: {
		url: 'https://262.ecma-international.org/14.0/#sec-atomicreadmodifywrite'
	},
	Await: {
		url: 'https://262.ecma-international.org/14.0/#await'
	},
	BackreferenceMatcher: {
		url: 'https://262.ecma-international.org/14.0/#sec-backreference-matcher'
	},
	'BigInt::add': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-add'
	},
	'BigInt::bitwiseAND': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-bitwiseAND'
	},
	'BigInt::bitwiseNOT': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-bitwiseNOT'
	},
	'BigInt::bitwiseOR': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-bitwiseOR'
	},
	'BigInt::bitwiseXOR': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-bitwiseXOR'
	},
	'BigInt::divide': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-divide'
	},
	'BigInt::equal': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-equal'
	},
	'BigInt::exponentiate': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-exponentiate'
	},
	'BigInt::leftShift': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-leftShift'
	},
	'BigInt::lessThan': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-lessThan'
	},
	'BigInt::multiply': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-multiply'
	},
	'BigInt::remainder': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-remainder'
	},
	'BigInt::signedRightShift': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-signedRightShift'
	},
	'BigInt::subtract': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-subtract'
	},
	'BigInt::toString': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-tostring'
	},
	'BigInt::unaryMinus': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-unaryMinus'
	},
	'BigInt::unsignedRightShift': {
		url: 'https://262.ecma-international.org/14.0/#sec-numeric-types-bigint-unsignedRightShift'
	},
	BigIntBitwiseOp: {
		url: 'https://262.ecma-international.org/14.0/#sec-bigintbitwiseop'
	},
	BinaryAnd: {
		url: 'https://262.ecma-international.org/14.0/#sec-binaryand'
	},
	BinaryOr: {
		url: 'https://262.ecma-international.org/14.0/#sec-binaryor'
	},
	BinaryXor: {
		url: 'https://262.ecma-international.org/14.0/#sec-binaryxor'
	},
	BlockDeclarationInstantiation: {
		url: 'https://262.ecma-international.org/14.0/#sec-blockdeclarationinstantiation'
	},
	BoundFunctionCreate: {
		url: 'https://262.ecma-international.org/14.0/#sec-boundfunctioncreate'
	},
	ByteListBitwiseOp: {
		url: 'https://262.ecma-international.org/14.0/#sec-bytelistbitwiseop'
	},
	ByteListEqual: {
		url: 'https://262.ecma-international.org/14.0/#sec-bytelistequal'
	},
	Call: {
		url: 'https://262.ecma-international.org/14.0/#sec-call'
	},
	CanBeHeldWeakly: {
		url: 'https://262.ecma-international.org/14.0/#sec-canbeheldweakly'
	},
	Canonicalize: {
		url: 'https://262.ecma-international.org/14.0/#sec-runtime-semantics-canonicalize-ch'
	},
	CanonicalNumericIndexString: {
		url: 'https://262.ecma-international.org/14.0/#sec-canonicalnumericindexstring'
	},
	CaseClauseIsSelected: {
		url: 'https://262.ecma-international.org/14.0/#sec-runtime-semantics-caseclauseisselected'
	},
	CharacterRange: {
		url: 'https://262.ecma-international.org/14.0/#sec-runtime-semantics-characterrange-abstract-operation'
	},
	CharacterRangeOrUnion: {
		url: 'https://262.ecma-international.org/14.0/#sec-runtime-semantics-characterrangeorunion-abstract-operation'
	},
	CharacterSetMatcher: {
		url: 'https://262.ecma-international.org/14.0/#sec-runtime-semantics-charactersetmatcher-abstract-operation'
	},
	clamp: {
		url: 'https://262.ecma-international.org/14.0/#clamping'
	},
	CleanupFinalizationRegistry: {
		url: 'https://262.ecma-international.org/14.0/#sec-cleanup-finalization-registry'
	},
	ClearKeptObjects: {
		url: 'https://262.ecma-international.org/14.0/#sec-clear-kept-objects'
	},
	CloneArrayBuffer: {
		url: 'https://262.ecma-international.org/14.0/#sec-clonearraybuffer'
	},
	CodePointAt: {
		url: 'https://262.ecma-international.org/14.0/#sec-codepointat'
	},
	CodePointsToString: {
		url: 'https://262.ecma-international.org/14.0/#sec-codepointstostring'
	},
	CompareArrayElements: {
		url: 'https://262.ecma-international.org/14.0/#sec-comparearrayelements'
	},
	CompareTypedArrayElements: {
		url: 'https://262.ecma-international.org/14.0/#sec-comparetypedarrayelements'
	},
	CompletePropertyDescriptor: {
		url: 'https://262.ecma-international.org/14.0/#sec-completepropertydescriptor'
	},
	Completion: {
		url: 'https://262.ecma-international.org/14.0/#sec-completion-ao'
	},
	CompletionRecord: {
		url: 'https://262.ecma-international.org/14.0/#sec-completion-record-specification-type'
	},
	ComposeWriteEventBytes: {
		url: 'https://262.ecma-international.org/14.0/#sec-composewriteeventbytes'
	},
	Construct: {
		url: 'https://262.ecma-international.org/14.0/#sec-construct'
	},
	ContinueDynamicImport: {
		url: 'https://262.ecma-international.org/14.0/#sec-ContinueDynamicImport'
	},
	ContinueModuleLoading: {
		url: 'https://262.ecma-international.org/14.0/#sec-ContinueModuleLoading'
	},
	CopyDataBlockBytes: {
		url: 'https://262.ecma-international.org/14.0/#sec-copydatablockbytes'
	},
	CopyDataProperties: {
		url: 'https://262.ecma-international.org/14.0/#sec-copydataproperties'
	},
	CountLeftCapturingParensBefore: {
		url: 'https://262.ecma-international.org/14.0/#sec-countleftcapturingparensbefore'
	},
	CountLeftCapturingParensWithin: {
		url: 'https://262.ecma-international.org/14.0/#sec-countleftcapturingparenswithin'
	},
	CreateArrayFromList: {
		url: 'https://262.ecma-international.org/14.0/#sec-createarrayfromlist'
	},
	CreateArrayIterator: {
		url: 'https://262.ecma-international.org/14.0/#sec-createarrayiterator'
	},
	CreateAsyncFromSyncIterator: {
		url: 'https://262.ecma-international.org/14.0/#sec-createasyncfromsynciterator'
	},
	CreateAsyncIteratorFromClosure: {
		url: 'https://262.ecma-international.org/14.0/#sec-createasynciteratorfromclosure'
	},
	CreateBuiltinFunction: {
		url: 'https://262.ecma-international.org/14.0/#sec-createbuiltinfunction'
	},
	CreateByteDataBlock: {
		url: 'https://262.ecma-international.org/14.0/#sec-createbytedatablock'
	},
	CreateDataProperty: {
		url: 'https://262.ecma-international.org/14.0/#sec-createdataproperty'
	},
	CreateDataPropertyOrThrow: {
		url: 'https://262.ecma-international.org/14.0/#sec-createdatapropertyorthrow'
	},
	CreateDynamicFunction: {
		url: 'https://262.ecma-international.org/14.0/#sec-createdynamicfunction'
	},
	CreateForInIterator: {
		url: 'https://262.ecma-international.org/14.0/#sec-createforiniterator'
	},
	CreateHTML: {
		url: 'https://262.ecma-international.org/14.0/#sec-createhtml'
	},
	CreateIntrinsics: {
		url: 'https://262.ecma-international.org/14.0/#sec-createintrinsics'
	},
	CreateIteratorFromClosure: {
		url: 'https://262.ecma-international.org/14.0/#sec-createiteratorfromclosure'
	},
	CreateIterResultObject: {
		url: 'https://262.ecma-international.org/14.0/#sec-createiterresultobject'
	},
	CreateListFromArrayLike: {
		url: 'https://262.ecma-international.org/14.0/#sec-createlistfromarraylike'
	},
	CreateListIteratorRecord: {
		url: 'https://262.ecma-international.org/14.0/#sec-createlistiteratorRecord'
	},
	CreateMapIterator: {
		url: 'https://262.ecma-international.org/14.0/#sec-createmapiterator'
	},
	CreateMappedArgumentsObject: {
		url: 'https://262.ecma-international.org/14.0/#sec-createmappedargumentsobject'
	},
	CreateMethodProperty: {
		url: 'https://262.ecma-international.org/14.0/#sec-createmethodproperty'
	},
	CreateNonEnumerableDataPropertyOrThrow: {
		url: 'https://262.ecma-international.org/14.0/#sec-createnonenumerabledatapropertyorthrow'
	},
	CreatePerIterationEnvironment: {
		url: 'https://262.ecma-international.org/14.0/#sec-createperiterationenvironment'
	},
	CreateRealm: {
		url: 'https://262.ecma-international.org/14.0/#sec-createrealm'
	},
	CreateRegExpStringIterator: {
		url: 'https://262.ecma-international.org/14.0/#sec-createregexpstringiterator'
	},
	CreateResolvingFunctions: {
		url: 'https://262.ecma-international.org/14.0/#sec-createresolvingfunctions'
	},
	CreateSetIterator: {
		url: 'https://262.ecma-international.org/14.0/#sec-createsetiterator'
	},
	CreateSharedByteDataBlock: {
		url: 'https://262.ecma-international.org/14.0/#sec-createsharedbytedatablock'
	},
	CreateUnmappedArgumentsObject: {
		url: 'https://262.ecma-international.org/14.0/#sec-createunmappedargumentsobject'
	},
	DateFromTime: {
		url: 'https://262.ecma-international.org/14.0/#sec-date-number'
	},
	DateString: {
		url: 'https://262.ecma-international.org/14.0/#sec-datestring'
	},
	Day: {
		url: 'https://262.ecma-international.org/14.0/#eqn-Day'
	},
	DayFromYear: {
		url: 'https://262.ecma-international.org/14.0/#eqn-DaysFromYear'
	},
	DaysInYear: {
		url: 'https://262.ecma-international.org/14.0/#eqn-DaysInYear'
	},
	DayWithinYear: {
		url: 'https://262.ecma-international.org/14.0/#eqn-DayWithinYear'
	},
	Decode: {
		url: 'https://262.ecma-international.org/14.0/#sec-decode'
	},
	DefaultTimeZone: {
		url: 'https://262.ecma-international.org/14.0/#sec-defaulttimezone'
	},
	DefineField: {
		url: 'https://262.ecma-international.org/14.0/#sec-definefield'
	},
	DefineMethodProperty: {
		url: 'https://262.ecma-international.org/14.0/#sec-definemethodproperty'
	},
	DefinePropertyOrThrow: {
		url: 'https://262.ecma-international.org/14.0/#sec-definepropertyorthrow'
	},
	DeletePropertyOrThrow: {
		url: 'https://262.ecma-international.org/14.0/#sec-deletepropertyorthrow'
	},
	DetachArrayBuffer: {
		url: 'https://262.ecma-international.org/14.0/#sec-detacharraybuffer'
	},
	Encode: {
		url: 'https://262.ecma-international.org/14.0/#sec-encode'
	},
	EnterCriticalSection: {
		url: 'https://262.ecma-international.org/14.0/#sec-entercriticalsection'
	},
	EnumerableOwnProperties: {
		url: 'https://262.ecma-international.org/14.0/#sec-enumerableownproperties'
	},
	EnumerateObjectProperties: {
		url: 'https://262.ecma-international.org/14.0/#sec-enumerate-object-properties'
	},
	EscapeRegExpPattern: {
		url: 'https://262.ecma-international.org/14.0/#sec-escaperegexppattern'
	},
	EvalDeclarationInstantiation: {
		url: 'https://262.ecma-international.org/14.0/#sec-evaldeclarationinstantiation'
	},
	EvaluateCall: {
		url: 'https://262.ecma-international.org/14.0/#sec-evaluatecall'
	},
	EvaluateNew: {
		url: 'https://262.ecma-international.org/14.0/#sec-evaluatenew'
	},
	EvaluatePropertyAccessWithExpressionKey: {
		url: 'https://262.ecma-international.org/14.0/#sec-evaluate-property-access-with-expression-key'
	},
	EvaluatePropertyAccessWithIdentifierKey: {
		url: 'https://262.ecma-international.org/14.0/#sec-evaluate-property-access-with-identifier-key'
	},
	EvaluateStringOrNumericBinaryExpression: {
		url: 'https://262.ecma-international.org/14.0/#sec-evaluatestringornumericbinaryexpression'
	},
	EventSet: {
		url: 'https://262.ecma-international.org/14.0/#sec-event-set'
	},
	ExecuteAsyncModule: {
		url: 'https://262.ecma-international.org/14.0/#sec-execute-async-module'
	},
	'𝔽': {
		url: 'https://262.ecma-international.org/14.0/#𝔽'
	},
	FindViaPredicate: {
		url: 'https://262.ecma-international.org/14.0/#sec-findviapredicate'
	},
	FinishLoadingImportedModule: {
		url: 'https://262.ecma-international.org/14.0/#sec-FinishLoadingImportedModule'
	},
	FlattenIntoArray: {
		url: 'https://262.ecma-international.org/14.0/#sec-flattenintoarray'
	},
	floor: {
		url: 'https://262.ecma-international.org/14.0/#eqn-floor'
	},
	ForBodyEvaluation: {
		url: 'https://262.ecma-international.org/14.0/#sec-forbodyevaluation'
	},
	'ForIn/OfBodyEvaluation': {
		url: 'https://262.ecma-international.org/14.0/#sec-runtime-semantics-forin-div-ofbodyevaluation-lhs-stmt-iterator-lhskind-labelset'
	},
	'ForIn/OfHeadEvaluation': {
		url: 'https://262.ecma-international.org/14.0/#sec-runtime-semantics-forinofheadevaluation'
	},
	FromPropertyDescriptor: {
		url: 'https://262.ecma-international.org/14.0/#sec-frompropertydescriptor'
	},
	FulfillPromise: {
		url: 'https://262.ecma-international.org/14.0/#sec-fulfillpromise'
	},
	FunctionDeclarationInstantiation: {
		url: 'https://262.ecma-international.org/14.0/#sec-functiondeclarationinstantiation'
	},
	GatherAvailableAncestors: {
		url: 'https://262.ecma-international.org/14.0/#sec-gather-available-ancestors'
	},
	GeneratorResume: {
		url: 'https://262.ecma-international.org/14.0/#sec-generatorresume'
	},
	GeneratorResumeAbrupt: {
		url: 'https://262.ecma-international.org/14.0/#sec-generatorresumeabrupt'
	},
	GeneratorStart: {
		url: 'https://262.ecma-international.org/14.0/#sec-generatorstart'
	},
	GeneratorValidate: {
		url: 'https://262.ecma-international.org/14.0/#sec-generatorvalidate'
	},
	GeneratorYield: {
		url: 'https://262.ecma-international.org/14.0/#sec-generatoryield'
	},
	Get: {
		url: 'https://262.ecma-international.org/14.0/#sec-get-o-p'
	},
	GetActiveScriptOrModule: {
		url: 'https://262.ecma-international.org/14.0/#sec-getactivescriptormodule'
	},
	GetFunctionRealm: {
		url: 'https://262.ecma-international.org/14.0/#sec-getfunctionrealm'
	},
	GetGeneratorKind: {
		url: 'https://262.ecma-international.org/14.0/#sec-getgeneratorkind'
	},
	GetGlobalObject: {
		url: 'https://262.ecma-international.org/14.0/#sec-getglobalobject'
	},
	GetIdentifierReference: {
		url: 'https://262.ecma-international.org/14.0/#sec-getidentifierreference'
	},
	GetImportedModule: {
		url: 'https://262.ecma-international.org/14.0/#sec-GetImportedModule'
	},
	GetIterator: {
		url: 'https://262.ecma-international.org/14.0/#sec-getiterator'
	},
	GetIteratorFromMethod: {
		url: 'https://262.ecma-international.org/14.0/#sec-getiteratorfrommethod'
	},
	GetMatchIndexPair: {
		url: 'https://262.ecma-international.org/14.0/#sec-getmatchindexpair'
	},
	GetMatchString: {
		url: 'https://262.ecma-international.org/14.0/#sec-getmatchstring'
	},
	GetMethod: {
		url: 'https://262.ecma-international.org/14.0/#sec-getmethod'
	},
	GetModifySetValueInBuffer: {
		url: 'https://262.ecma-international.org/14.0/#sec-getmodifysetvalueinbuffer'
	},
	GetModuleNamespace: {
		url: 'https://262.ecma-international.org/14.0/#sec-getmodulenamespace'
	},
	GetNamedTimeZoneEpochNanoseconds: {
		url: 'https://262.ecma-international.org/14.0/#sec-getnamedtimezoneepochnanoseconds'
	},
	GetNamedTimeZoneOffsetNanoseconds: {
		url: 'https://262.ecma-international.org/14.0/#sec-getnamedtimezoneoffsetnanoseconds'
	},
	GetNewTarget: {
		url: 'https://262.ecma-international.org/14.0/#sec-getnewtarget'
	},
	GetOwnPropertyKeys: {
		url: 'https://262.ecma-international.org/14.0/#sec-getownpropertykeys'
	},
	GetPromiseResolve: {
		url: 'https://262.ecma-international.org/14.0/#sec-getpromiseresolve'
	},
	GetPrototypeFromConstructor: {
		url: 'https://262.ecma-international.org/14.0/#sec-getprototypefromconstructor'
	},
	GetStringIndex: {
		url: 'https://262.ecma-international.org/14.0/#sec-getstringindex'
	},
	GetSubstitution: {
		url: 'https://262.ecma-international.org/14.0/#sec-getsubstitution'
	},
	GetSuperConstructor: {
		url: 'https://262.ecma-international.org/14.0/#sec-getsuperconstructor'
	},
	GetTemplateObject: {
		url: 'https://262.ecma-international.org/14.0/#sec-gettemplateobject'
	},
	GetThisEnvironment: {
		url: 'https://262.ecma-international.org/14.0/#sec-getthisenvironment'
	},
	GetThisValue: {
		url: 'https://262.ecma-international.org/14.0/#sec-getthisvalue'
	},
	GetUTCEpochNanoseconds: {
		url: 'https://262.ecma-international.org/14.0/#sec-getutcepochnanoseconds'
	},
	GetV: {
		url: 'https://262.ecma-international.org/14.0/#sec-getv'
	},
	GetValue: {
		url: 'https://262.ecma-international.org/14.0/#sec-getvalue'
	},
	GetValueFromBuffer: {
		url: 'https://262.ecma-international.org/14.0/#sec-getvaluefrombuffer'
	},
	GetViewValue: {
		url: 'https://262.ecma-international.org/14.0/#sec-getviewvalue'
	},
	GetWaiterList: {
		url: 'https://262.ecma-international.org/14.0/#sec-getwaiterlist'
	},
	GlobalDeclarationInstantiation: {
		url: 'https://262.ecma-international.org/14.0/#sec-globaldeclarationinstantiation'
	},
	GroupSpecifiersThatMatch: {
		url: 'https://262.ecma-international.org/14.0/#sec-groupspecifiersthatmatch'
	},
	'happens-before': {
		url: 'https://262.ecma-international.org/14.0/#sec-happens-before'
	},
	HasOwnProperty: {
		url: 'https://262.ecma-international.org/14.0/#sec-hasownproperty'
	},
	HasProperty: {
		url: 'https://262.ecma-international.org/14.0/#sec-hasproperty'
	},
	'host-synchronizes-with': {
		url: 'https://262.ecma-international.org/14.0/#sec-host-synchronizes-with'
	},
	HostEventSet: {
		url: 'https://262.ecma-international.org/14.0/#sec-hosteventset'
	},
	HourFromTime: {
		url: 'https://262.ecma-international.org/14.0/#eqn-HourFromTime'
	},
	IfAbruptCloseIterator: {
		url: 'https://262.ecma-international.org/14.0/#sec-ifabruptcloseiterator'
	},
	IfAbruptRejectPromise: {
		url: 'https://262.ecma-international.org/14.0/#sec-ifabruptrejectpromise'
	},
	ImportedLocalNames: {
		url: 'https://262.ecma-international.org/14.0/#sec-importedlocalnames'
	},
	InitializeBoundName: {
		url: 'https://262.ecma-international.org/14.0/#sec-initializeboundname'
	},
	InitializeHostDefinedRealm: {
		url: 'https://262.ecma-international.org/14.0/#sec-initializehostdefinedrealm'
	},
	InitializeInstanceElements: {
		url: 'https://262.ecma-international.org/14.0/#sec-initializeinstanceelements'
	},
	InitializeReferencedBinding: {
		url: 'https://262.ecma-international.org/14.0/#sec-initializereferencedbinding'
	},
	InitializeTypedArrayFromArrayBuffer: {
		url: 'https://262.ecma-international.org/14.0/#sec-initializetypedarrayfromarraybuffer'
	},
	InitializeTypedArrayFromArrayLike: {
		url: 'https://262.ecma-international.org/14.0/#sec-initializetypedarrayfromarraylike'
	},
	InitializeTypedArrayFromList: {
		url: 'https://262.ecma-international.org/14.0/#sec-initializetypedarrayfromlist'
	},
	InitializeTypedArrayFromTypedArray: {
		url: 'https://262.ecma-international.org/14.0/#sec-initializetypedarrayfromtypedarray'
	},
	InLeapYear: {
		url: 'https://262.ecma-international.org/14.0/#eqn-InLeapYear'
	},
	InnerModuleEvaluation: {
		url: 'https://262.ecma-international.org/14.0/#sec-innermoduleevaluation'
	},
	InnerModuleLinking: {
		url: 'https://262.ecma-international.org/14.0/#sec-InnerModuleLinking'
	},
	InnerModuleLoading: {
		url: 'https://262.ecma-international.org/14.0/#sec-InnerModuleLoading'
	},
	InstallErrorCause: {
		url: 'https://262.ecma-international.org/14.0/#sec-installerrorcause'
	},
	InstanceofOperator: {
		url: 'https://262.ecma-international.org/14.0/#sec-instanceofoperator'
	},
	IntegerIndexedElementGet: {
		url: 'https://262.ecma-international.org/14.0/#sec-integerindexedelementget'
	},
	IntegerIndexedElementSet: {
		url: 'https://262.ecma-international.org/14.0/#sec-integerindexedelementset'
	},
	IntegerIndexedObjectCreate: {
		url: 'https://262.ecma-international.org/14.0/#sec-integerindexedobjectcreate'
	},
	InternalizeJSONProperty: {
		url: 'https://262.ecma-international.org/14.0/#sec-internalizejsonproperty'
	},
	Invoke: {
		url: 'https://262.ecma-international.org/14.0/#sec-invoke'
	},
	IsAccessorDescriptor: {
		url: 'https://262.ecma-international.org/14.0/#sec-isaccessordescriptor'
	},
	IsAnonymousFunctionDefinition: {
		url: 'https://262.ecma-international.org/14.0/#sec-isanonymousfunctiondefinition'
	},
	IsArray: {
		url: 'https://262.ecma-international.org/14.0/#sec-isarray'
	},
	IsBigIntElementType: {
		url: 'https://262.ecma-international.org/14.0/#sec-isbigintelementtype'
	},
	IsCallable: {
		url: 'https://262.ecma-international.org/14.0/#sec-iscallable'
	},
	IsCompatiblePropertyDescriptor: {
		url: 'https://262.ecma-international.org/14.0/#sec-iscompatiblepropertydescriptor'
	},
	IsConcatSpreadable: {
		url: 'https://262.ecma-international.org/14.0/#sec-isconcatspreadable'
	},
	IsConstructor: {
		url: 'https://262.ecma-international.org/14.0/#sec-isconstructor'
	},
	IsDataDescriptor: {
		url: 'https://262.ecma-international.org/14.0/#sec-isdatadescriptor'
	},
	IsDetachedBuffer: {
		url: 'https://262.ecma-international.org/14.0/#sec-isdetachedbuffer'
	},
	IsExtensible: {
		url: 'https://262.ecma-international.org/14.0/#sec-isextensible-o'
	},
	IsGenericDescriptor: {
		url: 'https://262.ecma-international.org/14.0/#sec-isgenericdescriptor'
	},
	IsInTailPosition: {
		url: 'https://262.ecma-international.org/14.0/#sec-isintailposition'
	},
	IsIntegralNumber: {
		url: 'https://262.ecma-international.org/14.0/#sec-isintegralnumber'
	},
	IsLabelledFunction: {
		url: 'https://262.ecma-international.org/14.0/#sec-islabelledfunction'
	},
	IsLessThan: {
		url: 'https://262.ecma-international.org/14.0/#sec-islessthan'
	},
	IsLooselyEqual: {
		url: 'https://262.ecma-international.org/14.0/#sec-islooselyequal'
	},
	IsNoTearConfiguration: {
		url: 'https://262.ecma-international.org/14.0/#sec-isnotearconfiguration'
	},
	IsPrivateReference: {
		url: 'https://262.ecma-international.org/14.0/#sec-isprivatereference'
	},
	IsPromise: {
		url: 'https://262.ecma-international.org/14.0/#sec-ispromise'
	},
	IsPropertyKey: {
		url: 'https://262.ecma-international.org/14.0/#sec-ispropertykey'
	},
	IsPropertyReference: {
		url: 'https://262.ecma-international.org/14.0/#sec-ispropertyreference'
	},
	IsRegExp: {
		url: 'https://262.ecma-international.org/14.0/#sec-isregexp'
	},
	IsSharedArrayBuffer: {
		url: 'https://262.ecma-international.org/14.0/#sec-issharedarraybuffer'
	},
	IsStrictlyEqual: {
		url: 'https://262.ecma-international.org/14.0/#sec-isstrictlyequal'
	},
	IsStringWellFormedUnicode: {
		url: 'https://262.ecma-international.org/14.0/#sec-isstringwellformedunicode'
	},
	IsSuperReference: {
		url: 'https://262.ecma-international.org/14.0/#sec-issuperreference'
	},
	IsTimeZoneOffsetString: {
		url: 'https://262.ecma-international.org/14.0/#sec-istimezoneoffsetstring'
	},
	IsUnclampedIntegerElementType: {
		url: 'https://262.ecma-international.org/14.0/#sec-isunclampedintegerelementtype'
	},
	IsUnresolvableReference: {
		url: 'https://262.ecma-international.org/14.0/#sec-isunresolvablereference'
	},
	IsUnsignedElementType: {
		url: 'https://262.ecma-international.org/14.0/#sec-isunsignedelementtype'
	},
	IsValidIntegerIndex: {
		url: 'https://262.ecma-international.org/14.0/#sec-isvalidintegerindex'
	},
	IsValidRegularE
